spk/sonarr/Makefile

```diff
@@ -35,6 +35,11 @@ POST_STRIP_TARGET = sonarr_extra_install
 
 include ../../mk/spksrc.spk.mk
 
+PACKAGE_VERSION = $(SPK_NAME_ARCH)-$(SPK_TCVERS)_$(SPK_VERS)-$(SPK_REV)
+PACKAGE_AUTHOR = [Team Sonarr](https://sonarr.tv)
```

Comment:
I was just going based on these examples:
https://github.com/SynoCommunity/spksrc/blob/44a3d5d31002250a5fa66281e392f671275e2e95/spk/lidarr/Makefile#L39-L40
https://github.com/SynoCommunity/spksrc/blob/44a3d5d31002250a5fa66281e392f671275e2e95/spk/radarr/Makefile#L43-L44
So I'm guessing they were going for author and nor maintainer?

Reply:
So what would you propose? Something like this?
```
PACKAGE_AUTHOR = [SynoCommunity](https://synocommunity.com)
```

Reply:
That's up to SynoComm

Reply:
@hgy59, can you weigh in on this please? Or tag whomever else should comment on this.

Reply:
I've reviewed how this presents in the interface and I believe my proposal should be fine. Updated the PR.

Reply:
What's that screenshot look like? Since again the Sonarr team neither maintain, support, nor develop the package.

Reply:
While in the other ***arr** apps, the information in the `package_info` file seems to be added to the system status screen, the code to pull this information does not seem to be present in Sonarr. As such, the screenshot looks no different with or without this file... Here's what it looks like from this PR:

<img width="484" alt="Screenshot 2022-12-07 at 5 29 31 PM" src="https://user-images.githubusercontent.com/943378/206300522-18edac31-d1f2-4c0d-a28d-294f5eef4b32.png">

And here's what the other packages look like that have this code:

<img width="492" alt="Screenshot 2022-12-07 at 5 30 52 PM" src="https://user-images.githubusercontent.com/943378/206300756-29636f37-0a57-4592-92d5-50294a75bd20.png">
<img width="482" alt="Screenshot 2022-12-07 at 5 31 00 PM" src="https://user-images.githubusercontent.com/943378/206300766-2b5c59eb-f33d-4aa4-8606-97c35f64ff66.png">

Reply:
That info comes from the package_info file.

What does the Syno Package itself look like in store / when adding or whatever?

Reply:
The only other place I see that code is perhaps in the installation screen like this:

<img width="689" alt="Screenshot 2022-12-07 at 7 59 05 PM" src="https://user-images.githubusercontent.com/943378/206323273-17a485a6-ae2f-4d5b-81dd-cfc1995cf654.png">

This would be different than with another package like Lidarr:

<img width="686" alt="Screenshot 2022-12-07 at 8 00 03 PM" src="https://user-images.githubusercontent.com/943378/206323303-be2ad70b-3098-42e2-a1e2-b717081fa8ff.png">

Reply:
But to be fair, the screen looks the same as the Sonarr currently in the SynoCommunity repository:

<img width="689" alt="Screenshot 2022-12-07 at 8 03 34 PM" src="https://user-images.githubusercontent.com/943378/206323740-1155be89-5e58-49f3-8bf4-02a1b3389452.png">

So I'm not really sure where the `package_info` would make a difference.